{
  "affected_release": [
    {
      "product_name": "Red Hat Ansible Engine 2.9 for RHEL 7",
      "release_date": "2020-03-11T00:00:00Z",
      "advisory": "RHBA-2020:0784",
      "package": "ansible-0:2.9.6-1.el7ae",
      "cpe": "cpe:/a:redhat:ansible_engine:2"
    },
    {
      "product_name": "Red Hat Ansible Engine 2.9 for RHEL 8",
      "release_date": "2020-03-11T00:00:00Z",
      "advisory": "RHBA-2020:0784",
      "package": "ansible-0:2.9.6-1.el8ae",
      "cpe": "cpe:/a:redhat:ansible_engine:2"
    }
  ],
  "package_state": null,
  "threat_severity": "Moderate",
  "public_date": "2017-12-21T00:00:00Z",
  "bugzilla": {
    "description": "CVE-2020-10729 Ansible: two random password lookups in same task return same value",
    "id": "1831089",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1831089"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "5.0",
    "cvss3_scoring_vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:R/S:U/C:H/I:N/A:N",
    "status": "verified"
  },
  "iava": "",
  "cwe": "CWE-330",
  "statement": "",
  "acknowledgement": "Red Hat would like to thank Rihards Olups for reporting this issue.",
  "name": "CVE-2020-10729",
  "document_distribution": "",
  "details": [
    "No description is available for this CVE."
  ],
  "references": null
}